samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+order: 5
+id: excel-culture-info-date-time
+name: 'Culture info: date and time'
+description: This sample shows how to apply the cultural settings APIs to retrieve read-only operating system date and time settings.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.12'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#display-date-time-setting").click(() => tryCatch(displayDateTimeSetting));
+        $("#write-date-time-setting").click(() => tryCatch(writeDateTimeSetting));
+
+        /* Create table with only header content. */
+        async function setup() {
+          await Excel.run(async (context) => {
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+            const range = sheet.getRange("A1:B6");
+            range.values = [["Culture Setting", "Setting Format"], ["", ""], ["", ""], ["", ""], ["", ""], ["", ""]];
```

Comment:
Instead of adding several empty cells, considering making the original range "A1:B1" so you only need to worry about the headers.